monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_linux_wget.js

```diff
@@ -1,10 +1,13 @@
 import {OS_TYPES} from '../utils/OsTypes';
 
 
-export default function generateLocalLinuxWget(ip, osType) {
+export default function generateLocalLinuxWget(ip, osType, username) {
     let bitText = osType === OS_TYPES.LINUX_32 ? '32' : '64';
-    return `wget --no-check-certificate https://${ip}:5000/api/monkey/download/
-    monkey-linux-${bitText};
-    chmod +x monkey-linux-${bitText};
-    ./monkey-linux-${bitText} m0nk3y -s ${ip}:5000`;
+    let command = `wget --no-check-certificate https://${ip}:5000/api/monkey/download/`
+      + `monkey-linux-${bitText}; `
+      + `chmod +x monkey-linux-${bitText}; `
+      + `./monkey-linux-${bitText} m0nk3y -s ${ip}:5000`;
+    if (username != '')
+      command = `su - ${username} -c "${command}"`;
```

Comment:
Use curly-braces for ifs. Easier to extend and read, even if it's a single sentence.

Reply:
Also, this is a code duplication with other linux run method.